components/TextFields/src/MDCTextField.m

```diff
@@ -76,15 +83,25 @@ - (void)dealloc {
 - (void)encodeWithCoder:(NSCoder *)aCoder {
   [super encodeWithCoder:aCoder];
   [aCoder encodeObject:self.fundament forKey:MDCTextFieldFundamentKey];
+  [aCoder encodeInteger:self.leftViewMode forKey:MDCTextFieldLeftViewModeKey];
+  [aCoder encodeInteger:self.rightViewMode forKey:MDCTextFieldRightViewModeKey];
 }
 
 - (instancetype)copyWithZone:(NSZone *)zone {
   MDCTextField *copy = [[[self class] alloc] init];
 
   copy.fundament = [self.fundament copy];
   copy.enabled = self.isEnabled;
+  if ([self.leadingView conformsToProtocol:@protocol(NSCopying)]) {
```

Comment:
The init is not overridden. When you call init on a UIView, it calls the designated initializer which is initWithFrame which calls the common init.

But to be even clearer, I've made it the initWithFrame explicitly instead of implicitly.